lib/cluster.js

```diff
@@ -728,6 +728,14 @@ function sendHelper(proc, message, handle, cb) {
 // to the callback but intercepts and redirects ACK messages.
 function internal(worker, cb) {
   return function(message, handle) {
+    if (message.cmd === 'NODE_WARN') {
```

Comment:
Couldn't you just forward them by hand if you want to handle them in the cluster master? This seems like we are forcing a policy on people that they may or may not want.